libs/dh/shared/assets/src/assets/b2c-custom-layouts/changepassword-unified.html

```diff
@@ -75,7 +75,9 @@
     </script>
 
     <script>
-      const path = SETTINGS.remoteResource.replace(SETTINGS.remoteResource.split("/").pop(), "");
+      const path = window.SETTINGS
+        ? SETTINGS.remoteResource.replace(SETTINGS.remoteResource.split("/").pop(), "")
+        : "";
```

Comment:
question: in what case the `window.SETTINGS` can end up `undefined`

Reply:
The path is "", made it like that for local testing, because locally we can resolve the files from a relative path